CONTRIBUTING.md

```diff
@@ -196,62 +194,62 @@ You can set the `DEBUG` environment variable to turn on debugging logs. E.g. `DE
+Vite aims to be fully usable as a dependency in a TypeScript project (e.g., it should provide proper typings for VitePress), and also in `vite.config.ts`. This means technically a dependency whose types are exposed needs to be part of `dependencies` instead of `devDependencies`. However, this also means we won't be able to bundle it.
 
-To get around this, we inline some of these dependencies' types in `packages/vite/types`. This way we can still expose the typing but bundle the dependency's source code.
+To get around this, we inline some of these dependencies' types in `packages/vite/types`. This way, we can still expose the typing but bundle the dependency's source code.
 
-Use `pnpm run check-dist-types` to check bundled types does not rely on types in `devDependencies`. If you are adding `dependencies`, make sure to configure `tsconfig.check.json`.
+Use `pnpm run check-dist-types` to check that the bundled types do not rely on types in `devDependencies`. If you are adding `dependencies`, make sure to configure `tsconfig.check.json`.
 
-### Think before adding yet another option
+### Think Before Adding Yet Another Option
 
-We already have many config options, and we should avoid fixing an issue by adding yet another one. Before adding an option, try to think about:
+We already have many config options, and we should avoid fixing an issue by adding yet another one. Before adding an option, consider whether the problem:
 
-- Whether the problem is really worth addressing
-- Whether the problem can be fixed with a smarter default
-- Whether the problem has workaround using existing options
-- Whether the problem can be addressed with a plugin instead
+- is really worth addressing
+- can be fixed with a smarter default
+- has workaround using existing options
+- can be addressed with a plugin instead
 
-## Docs translation contribution
+## Docs Translation Contribution
 
 If you would like to start a translation in your language, you are welcome to contribute! Please join [the #translations channel in Vite Land](https://chat.vitejs.dev) to discuss and coordinate with others.
 
-The english docs are embedded in the main Vite repo, to allow contributors to work on docs, tests and implementation in the same PR. Translations are done by forking the main repo.
+The English docs are embedded in the main Vite repo to enable contributors to work on docs, tests, and implementation in the same PR. Translations are done by forking the main repo.
 
-### How to start a translation repo
+### How To Start a Translation Repo
 
-1. In order to get all doc files, you first need to clone this repo in your personal account.
-2. Keep all the files in `docs/` and remove everything else.
+1. Fork this repo to your personal account.
```

Comment:
I'm not sure if we should recommend forking since the steps below shows removing `.git`, and pushing to a new repo

Reply:
Ok, I misunderstood the original steps.